projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/breve/features/events.ts

```diff
@@ -106,7 +106,7 @@ function handleMouseLeave() {
 
 	highlightTimeout = setTimeout( () => {
 		( dispatch( 'jetpack/ai-breve' ) as BreveDispatch ).setHighlightHover( false );
-	}, 100 );
+	}, 100 ) as unknown as number;
```

Comment:
This needs to be reverted, but for some reason I can't build `plugins/jetpack` without it - leaving this as a reminder here to do so before merging.